pkg/public/server.go

```diff
@@ -119,6 +135,34 @@ func (s *Server) RegisterOpenAPIHandler() {
 	log.Infof("Raw API JSON available at %s", swaggerFilesPath+"/superplane.swagger.json")
 }
 
+func (s *Server) RegisterWebRoutes(webBasePath string) {
+	// The web app routes are registered on the main router
+	log.Infof("Registering web routes with base path: %s", webBasePath)
+
+	s.Router.HandleFunc("/ws/{canvasId}", s.handleWebSocket)
+
+	// Check if we're in development mode
+	if os.Getenv("ENV") == "dev" {
```

Comment:
Here, you use `ENV`, but in the fixtures, you `APP_ENV`. We should probably use just one env var, right?